common/app/model/content/Atoms.scala

```diff
@@ -120,6 +121,10 @@ object MediaAtom extends common.Logging {
       mimeType = mediaAsset.mimeType)
   }
 
+  def isoDuration(seconds: Long): String = {
```

Comment:
I would add this method in the `MediaAtom` class rather than here in the object.
You will be able to do:
```
myMediaAtom.isoDuration
```
vs
```
MediaAtom.isoDuration(myMediaAtom.duration)

```